doc/LIMITS.md

```diff
@@ -75,25 +94,44 @@ const node = await createLibp2pNode({
   connectionManager: {
     /**
      * If the node transfers more than this amount of data in bytes/second
-     * low value connections may be closed
+     * connections to untagged peers or those not in the allow list may be
+     * closed.
+     *
+     * This field is optional, the default value is shown.
+     *
+     * It is bytes per second.
      */
     maxData: 1024 * 1024,
```

Comment:
Is this saying that if my sendRate + receiveRate  is greater than 1MB/S that connections might get closed? That seems pretty low for a default.

Reply:
Not sure where I got that figure from, the default is `Infinity`.  It's a weird limit tbh, surely you'd want the transfer speeds to be as fast as possible at all times?

If there's so much incoming data performance starts to suffer, the event loop delay detection should kick in.